nautobot_golden_config/models.py

```diff
@@ -129,22 +147,56 @@ def _verify_get_custom_compliance_data(compliance_details):
             raise ValidationError(VALIDATION_MSG.format(val, "String or Json", compliance_details[val]))
 
 
+def _get_hierconfig_remediation(obj):
+    """Returns the remediation plan."""
```

Comment:
"the remediation config snippet" or "remediating config"

Reply:
fixed in the next commit.